cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/HealthProjectType.java

```diff
@@ -0,0 +1,58 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static java.util.Arrays.stream;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.stream.Collectors;
+import nva.commons.core.SingletonCollector;
+
+public enum HealthProjectType {
+
+    DRUGSTUDY("DRUGSTUDY"),
```

Comment:
Why having a type in the enum at all? By default Jackson will use the enum name to serialise/deserialise to strings.